internal/controller/csiaddons/csiaddonsnode_controller.go

```diff
@@ -211,6 +213,10 @@ func (r *CSIAddonsNodeReconciler) resolveEndpoint(ctx context.Context, rawURL st
 	if err != nil {
 		return "", "", err
 	}
+	if r.EnableTLS {
+		// We need to use this name to accept certificates signed for pods
+		return podname, podname + "." + namespace + ".pod" + ":" + port, nil
```

Comment:
Is this going to work when we are using host networking?

Reply:
Removing this.